Makefile

```diff
@@ -64,21 +64,21 @@ test:
 setup-environment:
 	@if [ -e /usr/bin/yum ]; then \
 		scripts/rhel_setup_environment.sh; \
+	elif [ -e /usr/local/bin/brew ] || [ -e /opt/homebrew/bin/brew ]; then \
```

Comment:
Remove intel path from this if statement so that intel Mac's aren't supported by default. Add separate if statement highlighting intel macs aren't supported.

Reply:
Updated as suggested.